Source/carthage/Build.swift

```diff
@@ -258,6 +270,7 @@ extension BuildPlatform: ArgumentType {
 	private static let acceptedStrings: [String: BuildPlatform] = [
 		"Mac": .Mac, "macosx": .Mac,
 		"iOS": .iOS, "iphoneos": .iOS, "iphonesimulator": .iOS,
+		"watchOS": .watchOS, "watchos": .watchOS, "watchsimulator": .watchOS,
```

Comment:
These are compared case-insensitively, so the two variants of `watchos` aren't needed.